functions.php

```diff
@@ -188,10 +192,17 @@ function getMessage(): string {
     } return '';
 }
 
+
+/**
+ * Sanitises any potentially malicious entries before inserting into db
+ *
+ * @return array returns an array containing the sanitised character name and class name
+ */
 function sanitizePost(): array {
     $charname = filter_var($_POST['charname'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
     $class = filter_var($_POST['class'], FILTER_SANITIZE_FULL_SPECIAL_CHARS);
-    return [$charname, $class];
+    $image = filter_var($_POST['image'], FILTER_SANITIZE_URL);
+    return [$charname, $class, $image];
```

Comment:
if you use an associative array instead of an indexed array, it will be clearer what each piece of data is

Reply:
done and re-unit tested